apikey-service/src/main/java/eu/europeana/apikey/controller/ApikeyController.java

```diff
@@ -62,6 +65,21 @@
     private static final String APIKEYNOTREGISTERED = "Apikey %s is not registered";
     private static final String APIKEYMISSING = "Missing apikey in the header. Correct syntax: Authorization: APIKEY apikey";
     private static final String APIKEY_PATTERN = "APIKEY\\s+([^\\s]+)";
+    private static final String CAPTCHA_PATTERN = "Bearer\\s+([^\\s]+)";
+    private static final String CAPTCHA_MISSING = "Missing Captcha token in the header. Correct syntax: Authorization: Bearer CAPTCHA_TOKEN";
+    private static final String CAPTCHA_VERIFICATION_FAILED = "Captcha verification failed.";
+
+    @Value("${keycloak.manager-client-id}")
+    private String managerClientId;
+
+    @Value("${keycloak.manager-client-secret}")
+    private String managerClientSecret;
+
+    @Autowired
+    private CaptchaManager captchaManager;
+
+    @Autowired
+    private CustomKeycloakAuthenticationProvider customKeycloakAuthenticationProvider;
```

Comment:
It's not a big issue, but we try not to use field injection and instead use constructor-based (see also https://www.vojtechruzicka.com/field-dependency-injection-considered-harmful/)